05_iigo/index.tex

```diff
@@ -477,6 +477,40 @@ \subsection{End of Session Actions}
     \item The Judge has the \emph{power} to decide to hold an election for a new President.
 \end{enumerate}
 
+\section{Executive branch implementation}
+
+The executive branch has been implemented to facilitate the use of powers given to the president. Because any agent could potentially become the president, implementation of the executive branch has been split into two interconnected parts:
+\begin{itemize}
+\item Client-side implementation of presidential functions, which map directly to the powers given to the president.
+\item Server-side implementation of executive branch actions, which calls the presidential functions only on the currently residing president.
+\end{itemize}
+
+\subsection{Client-side implementation}
+\label{sub:president:client-side}
+Each agent can become a president therefore each agent has to implement all of the presidential functions according to the common interface. Those functions directly correspond to the powers of the executive branch outlined in Section~\ref{sec:executive} as well as accountability and transfer-of-power obligations outlined in Section~\ref{sec:accountability}. 
+
+The presidential functions can be divided into two categories:
+\begin{enumerate}
+\item Presidential power functions:
+    \begin{itemize}
+        \item \texttt{SetTaxationAmount} allows the president to decide the amount of individual taxation set to each island.
+        \item \texttt{EvaluateAllocationRequests} allows the president to decide the allocation of resources distributed from the common pool to the islands.
+        \item \texttt{PickRuleToVote} allows the president to select a rule for voting R∗ to be passed to the Speaker.
+    \end{itemize}
+\item The accountability cycle and transfer-of-power functions:
```

Comment:
```suggestion
\item The accountability cycle and transfer-of-power functions are the followings:
```

Reply:
there is a distinction in enumerate
1. Presidential power functions
2. The accountability cycle and transfer-of-power functions

so to keep the style 'are the following' is unnecesarry